server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/SetMessagesCreationProcessor.java

```diff
@@ -132,12 +132,14 @@ private void handleCreate(CreationMessageEntry create, Builder responseBuilder,
             assertIsUserOwnerOfMailboxes(mailboxIds, mailboxSession);
             performCreate(create, responseBuilder, mailboxSession);
         } catch (MailboxSendingNotAllowedException e) {
+            LOG.debug("{} is allowed to send a mail using {} identity", e.getConnectedUser().asString(), e.getFromField());
+
             responseBuilder.notCreated(create.getCreationId(),
                     SetError.builder()
                         .type(SetError.Type.INVALID_PROPERTIES)
                         .properties(MessageProperty.from)
-                        .description("Invalid 'from' field. Must be " +
-                                e.getAllowedFrom())
+                        .description("Invalid 'from' field. One accepted value is " +
+                                e.getConnectedUser().asString())
```

Comment:
are we leaking some information?

Reply:
In any case, not more than before.

So far, I bet that the connected user is known by the connected user so, no, we are not leaking information.